Mathlib/Tactic/Widget/StringDiagram.lean

```diff
@@ -361,4 +369,36 @@ open ProofWidgets
 def StringDiagram : Component PanelWidgetProps :=
   mk_rpc_widget% StringDiagram.rpc
 
+open Command
+
+/--
+Display the string diagram for a given term.
+
+Example usage:
+```
+/- String diagram for the equality theorem. We need `@` at the beginning of the theorem name. -/
+#string_diagram @MonoidalCategory.whisker_exchange
+
+/- String diagram for the morphism. -/
+variable {C : Type u} [Category.{v} C] [MonoidalCategory C] {X Y : C} (f : 𝟙_ C ⟶ X ⊗ Y) in
+#string_diagram f
+```
+-/
+syntax (name := stringDiagram) "#string_diagram " term : command
+
+@[command_elab stringDiagram, inherit_doc stringDiagram]
+def elabStringDiagramCmd : CommandElab := fun
+  | stx@`(#string_diagram $t:term) => do
+    let html ← runTermElabM fun _ => do
+      let e ← Term.elabTerm t none
```

Comment:
I think you can use something like `realizeGlobalConstNoOverloadWithInfo` instead of `elabTerm` to get the constant without requiring `@`.